src/components/Todo/List.jsx

```diff
@@ -1,13 +1,14 @@
 import Item from "./Item";
-import todoList from './dummyData'
 
-function List() {
+function List( props ) {
+  const { todoList } = props
 
   return (
     <div>
-      {todoList.map(item => (
+      {todoList.length > 0 && todoList.map(item => (
```

Comment:
funny enough, I don't think you need this `&&`, if the length is 0, the map function just won't fire